<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>KeyboardDialog</class>
 <widget class="QDialog" name="KeyboardDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>277</height>
   </rect>
  </property>
  <property name="font" >
   <font>
    <family>MS Shell Dlg 2</family>
    <pointsize>12</pointsize>
    <weight>75</weight>
    <italic>false</italic>
    <bold>true</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="windowTitle" >
   <string>Введите строку</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="buttonEsc" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>ESC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button1" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>10</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button2" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>10</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button3" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>10</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button4" >
   <property name="geometry" >
    <rect>
     <x>210</x>
     <y>10</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button5" >
   <property name="geometry" >
    <rect>
     <x>260</x>
     <y>10</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button6" >
   <property name="geometry" >
    <rect>
     <x>310</x>
     <y>10</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button7" >
   <property name="geometry" >
    <rect>
     <x>360</x>
     <y>10</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button8" >
   <property name="geometry" >
    <rect>
     <x>410</x>
     <y>10</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button9" >
   <property name="geometry" >
    <rect>
     <x>460</x>
     <y>10</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button0" >
   <property name="geometry" >
    <rect>
     <x>510</x>
     <y>10</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonMinus" >
   <property name="geometry" >
    <rect>
     <x>560</x>
     <y>10</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonPlus" >
   <property name="geometry" >
    <rect>
     <x>610</x>
     <y>10</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBack" >
   <property name="geometry" >
    <rect>
     <x>660</x>
     <y>10</y>
     <width>121</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>&lt;--</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonQ" >
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>60</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>q</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonW" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>60</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>w</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonE" >
   <property name="geometry" >
    <rect>
     <x>180</x>
     <y>60</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>e</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonR" >
   <property name="geometry" >
    <rect>
     <x>230</x>
     <y>60</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>r</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonT" >
   <property name="geometry" >
    <rect>
     <x>280</x>
     <y>60</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>t</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonY" >
   <property name="geometry" >
    <rect>
     <x>330</x>
     <y>60</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonU" >
   <property name="geometry" >
    <rect>
     <x>380</x>
     <y>60</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>u</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonI" >
   <property name="geometry" >
    <rect>
     <x>430</x>
     <y>60</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>i</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonO" >
   <property name="geometry" >
    <rect>
     <x>480</x>
     <y>60</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>o</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonP" >
   <property name="geometry" >
    <rect>
     <x>530</x>
     <y>60</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>p</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSlash" >
   <property name="geometry" >
    <rect>
     <x>580</x>
     <y>60</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBackslash" >
   <property name="geometry" >
    <rect>
     <x>630</x>
     <y>60</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>\</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonClear" >
   <property name="geometry" >
    <rect>
     <x>680</x>
     <y>60</y>
     <width>101</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>Очистить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonA" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>110</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>a</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonS" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>110</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>s</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonD" >
   <property name="geometry" >
    <rect>
     <x>210</x>
     <y>110</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>d</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonF" >
   <property name="geometry" >
    <rect>
     <x>260</x>
     <y>110</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>f</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonG" >
   <property name="geometry" >
    <rect>
     <x>310</x>
     <y>110</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>g</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonH" >
   <property name="geometry" >
    <rect>
     <x>360</x>
     <y>110</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>h</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonJ" >
   <property name="geometry" >
    <rect>
     <x>410</x>
     <y>110</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>j</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonK" >
   <property name="geometry" >
    <rect>
     <x>460</x>
     <y>110</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>k</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonL" >
   <property name="geometry" >
    <rect>
     <x>510</x>
     <y>110</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>l</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonColon" >
   <property name="geometry" >
    <rect>
     <x>560</x>
     <y>110</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSemicolon" >
   <property name="geometry" >
    <rect>
     <x>610</x>
     <y>110</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonEnter" >
   <property name="geometry" >
    <rect>
     <x>660</x>
     <y>110</y>
     <width>121</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>Ввод</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonLShift" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>160</y>
     <width>131</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>Shift</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonZ" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>160</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>z</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonX" >
   <property name="geometry" >
    <rect>
     <x>190</x>
     <y>160</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonC" >
   <property name="geometry" >
    <rect>
     <x>240</x>
     <y>160</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>c</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonV" >
   <property name="geometry" >
    <rect>
     <x>290</x>
     <y>160</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>v</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonB" >
   <property name="geometry" >
    <rect>
     <x>340</x>
     <y>160</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>b</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonN" >
   <property name="geometry" >
    <rect>
     <x>390</x>
     <y>160</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>n</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonM" >
   <property name="geometry" >
    <rect>
     <x>440</x>
     <y>160</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>m</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonComma" >
   <property name="geometry" >
    <rect>
     <x>490</x>
     <y>160</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>,</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDot" >
   <property name="geometry" >
    <rect>
     <x>540</x>
     <y>160</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonUnderbar" >
   <property name="geometry" >
    <rect>
     <x>590</x>
     <y>160</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>_</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonRShift" >
   <property name="geometry" >
    <rect>
     <x>640</x>
     <y>160</y>
     <width>141</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>Shift</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSpace" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>210</y>
     <width>461</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCaps" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>110</y>
     <width>101</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>Caps Lock</string>
   </property>
   <property name="checkable" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonRusLat" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>11</x>
     <y>60</y>
     <width>71</width>
     <height>52</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy>
     <hsizetype>5</hsizetype>
     <vsizetype>5</vsizetype>
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text" >
    <string>РУС</string>
   </property>
   <property name="checkable" >
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>buttonEsc</sender>
   <signal>clicked()</signal>
   <receiver>KeyboardDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>37</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel" >
     <x>56</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonEnter</sender>
   <signal>clicked()</signal>
   <receiver>KeyboardDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>686</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel" >
     <x>678</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
